game-core/src/main/java/games/strategy/engine/data/GameMap.java

```diff
@@ -207,6 +209,46 @@ public Territory getTerritory(final String s) {
     return getNeighbors(frontier, searched, distance, null);
   }
 
+  /**
+   * Returns all neighbors within a certain distance of the starting territory that territories
```

Comment:
The first sentence could use a re-read. Should it read something like the following (?): 

>  certain distance of the starting territory [where all territories] between the 2 match a condition.

Reply:
Agree. Updated.